lib/CallBackery/GuiPlugin/Abstract.pm

```diff
@@ -65,7 +66,7 @@ What should the tab holding this plugin be called
 =cut
 
 has 'tabName' => sub {
-    return shift->config->{'tab-name'};
+    return decode('utf8', shift->config->{'tab-name'});
 };
```

Comment:
this would indicate that the utf8 flag is  missing on the string stored in `shift->config->{'tab-name'};` the 'right' fix for this would then be to make sure the utf8 flag is set at the time when tab-name is loaded ...